lint/tests/test_lint.py

```diff
@@ -153,6 +153,58 @@ def test_lint_failing(capsys):
     assert err == ""
 
 
+def test_ref_existent_relative(capsys):
+    with _mock_lint("check_path") as mocked_check_path:
```

Comment:
Is mocking the best way to write those tests? `test_file_lints.py` might be a better place for them, and have less brittle patterns.